nni/retiarii/nn/pytorch/api.py

```diff
@@ -277,19 +287,25 @@ def forward(self, x):
     ----------
     candidates : list
         List of values to choose from.
+    prior : list of float
+        Prior distribution to sample from.
     label : str
         Identifier of the value choice.
     """
 
-    def __new__(cls, candidates: List[Any], label: Optional[str] = None):
+    # FIXME: prior is designed but not supported yet
+
+    def __new__(cls, candidates: List[Any], *, prior: Optional[List[float]] = None, label: Optional[str] = None):
```

Comment:
why only put `prior` in LayerChoice and ValueChoice, but not in `InputChoice`?

Reply:
Good question. Added it after reconsideration.